index.js

```diff
@@ -84,6 +84,9 @@ app.post("/containerize", createAppDir, upload.array("files"), async (req, res)
       }
     else if(framework == "Django"){
       port = 5000;
+    }
+    else if(framework == "Laravel"){
```

Comment:
@khalifan-kfan you forgot to specify the port for "Laravel-custom" framework, you could at it as an OR to the line that has "Laravel"